OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -119,6 +151,22 @@
     </codeblock>
 </div>
 
+<hr/>
+
+<h3 class="wiki-nav-item pb-2" id="CTL-x100-android-mode">
+    CTL-x100 is not detected by OpenTabletDriver
```

Comment:
Redundant to mention OpenTabletDriver here, we want titles as short as possible.
```suggestion
    CTL-x100 is not detected
```